validate/src/command.rs

```diff
@@ -128,13 +133,16 @@ impl Display for CommandValidationError {
 
                 f.write_str(" characters")
             }
-            CommandValidationErrorType::OptionNameInvalid => {
+            CommandValidationErrorType::OptionNameLengthInvalid => {
                 f.write_str("command option name must be between ")?;
-                Display::fmt(&OPTION_NAME_LENGTH_MIN, f)?;
+                Display::fmt(&NAME_LENGTH_MIN, f)?;
```

Comment:
we shouldn't change which const we use here, i believe

Reply:
The constant is used both to validate command name and option names, as they have the same limit.

Reply:
I know they're the same, I'd like to separate the constants anyway in case they do ever change.